src/main/resources/data.sql

```diff
@@ -45,4 +45,5 @@ INSERT INTO tutor_check_in(time_slot_assignment_id, time, date, remarks) VALUES
 INSERT INTO tutor_check_in(time_slot_assignment_id, time, date, remarks) VALUES (1, '6:00','9/3/20','explained recursion');
 INSERT INTO tutor_check_in(time_slot_assignment_id, time, date, remarks) VALUES (1, '10:00','5/10/20','helped a student with constructor method');
 
+INSERT INTO admin(email, is_permanent_admin) VALUES ('joe@ucsb.edu', FALSE);
```

Comment:
Consider `joe@example.com`... just in case there really is a `joe@ucsb.edu`.

There should never be an `example.com` (there is an RFC for that.)
https://tools.ietf.org/html/rfc6761